ups/lsst_ci.table

```diff
@@ -0,0 +1,12 @@
+setupRequired(lsst)
+setupRequired(lsst_apps)
+setupRequired(lsst_distrib)
+
+setupRequired(testdata_subaru)
+setupRequired(testdata_cfht)
+setupRequired(testdata_decam)
+setupRequired(obs_subaru)
+setupRequired(obs_cfht)
+setupRequired(obs_decam)
+
+setupRequired(obs_lsstSim)
```

Comment:
Good suggestion, thank you.  I had been forgetting about {{obs_sdss}}, but I agree that the intent is to also ensure that {{obs_sdss}} is being tested.

Done.